Source/WebKit/webpushd/ApplePushServiceConnection.mm

```diff
@@ -29,6 +29,7 @@
 #if HAVE(APPLE_PUSH_SERVICE_URL_TOKEN_SUPPORT)
 
 #import <wtf/BlockPtr.h>
+#import <wtf/WeakPtr.h>
```

Comment:
We need this for the build to work. `ApplePushServiceConnection::subscribe` uses `WeakPtr`. We were originally getting the header from `PushServiceConnection.h`, but no longer are. So I added it here.